File: krojac-ui/src/app/model/skirtRequest.ts
export class SkirtRequest {
  type!: string;     // A-line, Pencil, Circle
  waist!: number;    // waist circumference in cm
  hips!: number;     // hip circumference in cm
  length!: number;   // skirt length in cm

  constructor(init?: Partial<SkirtRequest>) {
    Object.assign(this, init);
  }
}

File: sbnz-integracija-projekta/service/src/main/java/com/ftn/sbnz/service/dto/SkirtRequest.java
package com.ftn.sbnz.service.dto;

public class SkirtRequest {
    private String type; // A-line, Pencil, Circle
    private int waist;
    private int hips;
    private int length;

    public String getType() {
        return type;
    }

    public void setType(String type) {
        this.type = type;
    }

    public int getWaist() {
        return waist;
    }

    public void setWaist(int waist) {
        this.waist = waist;
    }

    public int getHips() {
        return hips;
    }

    public void setHips(int hips) {
        this.hips = hips;
    }

    public int getLength() {
        return length;
    }

    public void setLength(int length) {
        this.length = length;
    }
}
